docs/kafka/manage-cluster/README.adoc

```diff
@@ -0,0 +1,250 @@
+--
++
+[NOTE]
+--
+These add-ons are required by and managed by {product-kafka} only and must not be adapted or changed in any way.
+--
++
+. The CLI lists the available clusters in your organization  and prompts you to select the ready cluster to register. Use the arrows on your keyboard to select the required cluster.
+. The CLI prompts you to create a machine pool with a specified node count. The required node count is calculated by identifying the number of requested Kafka streaming units and multiplying by three:
++
+----
+required_node_count = requested_Kafka_streaming_units × 3
+----
++
+For example, a Kafka instance of 2 streaming units requires a machine pool of 6 nodes. A streaming unit determines the default maximum capacity of a Kafka instance. An instance with a larger size can handle higher loads and process more events, has more storage, and can handle more clients and connections.
++
+If there is an existing machine pool which has the correct node labels and taints required by the Kafka Service Fleet Manager, then this machine pool is automatically selected for use by default. Labels and taints in a machine pool allow you to organize sets of objects and control the scheduling of pods. For more information about machine pools, labels, and taints, see https://access.redhat.com/documentation/en-us/red_hat_openshift_service_on_aws/4/html/cluster_administration/nodes#rosa-nodes-machinepools-about[About machine pools^].
++
+Enter `6` as the required node count to create a machine pool with 6 nodes called `kafka-standard`.
++
+[NOTE]
+--
+This kafka-standard machine pool is for use with {product-kafka} only. Do not assign your own production workloads to run within it.
+--
+. Enter `y` to confirm you want your Kafka instances to be accessible through a public network.
+
+.Verification
+
+. Verify the registration has been successful by logging in to the https://console.redhat.com/openshift[OpenShift Cluster Manager Hybrid Cloud Console^], clicking *Clusters*, and clicking the name of your cluster.
+. On your cluster page, click the *Machine pools* tab.
+.. Expand *kafka-standard* to view more details of the newly created machine pool including the labels and taints required by the Kafka Service Fleet Manager that have automatically been included as part of the registration process.
+. Click the *Add-ons* tab.
++
+On the *Add-ons* page, an installing icon appears, indicating that the services have begun installing. A green check mark appears in the service option when the installation is complete. You might have to refresh your browser to see the installation status.
+. (Optional) In the terminal, enter the list command to see your registered clusters with a ready state.
```

Comment:
This `list` command will show the status of the cluster, not just clusters in a ready state.

i.e. if the cluster is still waiting on an add-on to finish installation then the status would be `waiting for kas-fleet-shard operater` (or terminology of the same effect). If a cluster is in a ready state then that's what the status will show.